extensions/smallrye-reactive-messaging-pulsar/deployment/src/test/resources/application-secured.properties

```diff
@@ -0,0 +1,16 @@
+mp.messaging.outgoing.source.connector=smallrye-pulsar
+
+mp.messaging.incoming.in.connector=smallrye-pulsar
+mp.messaging.incoming.in.topic=source
+
+quarkus.pulsar.devservices.broker-config.authenticationEnabled=true
+quarkus.pulsar.devservices.broker-config.systemTopicEnabled=true
+quarkus.pulsar.devservices.broker-config.transactionCoordinatorEnabled=true
+quarkus.pulsar.devservices.broker-config.authenticationProviders=org.apache.pulsar.broker.authentication.AuthenticationProviderBasic
+quarkus.pulsar.devservices.broker-config.basicAuthConf=c3VwZXJ1c2VyOiRhcHIxJHpMOHY4VTZsJGNHTWdkckVja25RNHkzeC9ndWROajE=
+#quarkus.pulsar.devservices.broker-config.brokerClientAuthenticationEnabled=true
```

Comment:
should it be removed?

Reply:
I doesn't seem necessary for the authentication to work. But I'll keep it.